src/cljs/owlet/views/unsubscribe.cljs

```diff
@@ -19,8 +19,10 @@
 
 (defn unsubscribe-response [response]
   (cond
-    (= "Sent confirmation email." response) (reset! res 2)
-    (= "Not Subscribed." response) (reset! res 1)
+    (or
+      (= constants/confirmation-sent response)
+      (= constants/confirmation-resent response)) (reset! res 2)
```

Comment:
@ZadenRB I want to do a refactor around these  `(reset! res 1)` and  `(reset! res 2)` at some point too. Let me know if you can think of another way to do this without hardcoding integers.